src/components/Tour/TourEnd.tsx

```diff
@@ -0,0 +1,33 @@
+import { Flex, Text, useThemeContext } from "@radix-ui/themes";
+import { TourBox } from "./TourBox";
+import { TourTitle } from "./TourTitle";
+import { TourButton } from "./TourButton";
+import { useAppDispatch } from "../../hooks";
+import { finish } from "../../features/Tour";
+
+export const TourEnd = () => {
+  const appearance = useThemeContext().appearance;
```

Comment:
check the `hooks` directory for this, as the appearance can be changed by the body element (used in the Theme component)